spec/directions/bake_chapter_key_concepts_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterKeyConcepts do
+  before do
+    stub_locales({
+      'eoc_key_concepts': 'Key Concepts'
```

Comment:
Can you add `eoc_key_concepts` to the real `en.yml` translation file?

Reply:
it's added but on the `bake exercises` branch

Reply:
ok, we can't merge this one until that is in, and it is currently not as far along as this one?

Reply:
I'll add the change here and remove it from there, that PR might not make it that far